MmSupervisorPkg/Test/MmSupvRequestUnitTestApp/MmSupvRequestUnitTestApp.c

```diff
@@ -808,6 +808,7 @@ InspectSecurityPolicy (
 
   if (MemLevel == UNDEFINED_LEVEL) {

```

Comment:
Also, since you are already changing this, do you want to update the logic for TXT?

Should just be these 2 commits:
https://github.com/kuqin12/mu_feature_mm_supv/commit/cd66f00495eb51ec12f83e2ea3d76e410fdf8eac
https://github.com/kuqin12/mu_feature_mm_supv/commit/8beba97dee2c24956429d10524bff4013b1cb999

Reply:
I thought we didn't want to change the test when we talked yesterday?  If you think it's fine I'll add it here.

Reply:
Yeah, I think the existing test will try to check against some unintended pages like TPM and cause false positives. Let's change the test to check against TXT private and public register regions for 1 page each. (I think those above commits would do)

Reply:
I pulled in your commits that changed TXT.

Reply:
Thanks you!